src/pages/patterns/save.mdx

```diff
@@ -0,0 +1,258 @@
+3. In a [page footer](/components/chrome#footer), the Save and Cancel buttons remain fixed
+
+### Saving changes
+
+Saving applies to a page or view.
+
+Always [enable](/patterns/buttons#disabled) the Save button for users to trigger an action even if
+there are no new changes to save.
+
+- If there are tabs, save all data per tab
+- If the page or tab has an indication of the last edit date, that timestamp should update
+- After changes are saved, show a loader inside the Save button if saving takes time, and ignore
+  trigger actions for the duration
+- Ensure a custom, translated label for the loader is provided on implementation
+
+import SavePatternInteraction from '../../examples/save/SavePatternInteraction.tsx';
+import SavePatternInteractionCode from '!!raw-loader!../../examples/save/SavePatternInteraction.tsx';
+
+<CodeExample code={SavePatternInteractionCode}>
+  <SavePatternInteraction />
+</CodeExample>
+
+### Notifications
+
+#### Successful Save
+
+![Save pattern success notification](figma:patterns-save-anatomy-success-notification.png)
+
+1. A success [notification](/components/notifications) appears at the top right of the screen
+   indicating changes have been saved
+
+<BestPractice>
+  <Do
+    imageIsFreeForm
+    imageSource={props.data.saveSuccessDo.childFile.childImageSharp.gatsbyImageData}
+  >
+    <ul>
+      <li>Use transient notifications to confirm saving actions.</li>
+    </ul>
+  </Do>
+  <Dont
+    imageIsFreeForm
+    imageSource={props.data.saveSuccessDont.childFile.childImageSharp.gatsbyImageData}
+  >
+    <ul>
+      <li>Do not place actions inside notifications as it is transient and inaccessible.</li>
+    </ul>
+  </Dont>
+</BestPractice>
+
+## States
+
+### Save errors
+
+Errors can be presented in a variety of ways, depending on the issue.
+
+![Save pattern error single field](figma:patterns-save-errors.png)
+
+1. Show validation errors inline if the Save button is actioned but something is wrong
+
+![Save pattern multiple errors](figma:patterns-save-validation-alert.png)
+
+1. An error [alert](/components/alerts#type) that summarizes the errors at the top of the page improves
+   screen reader accessibility
+
+### Unsaved changes
+
+If a user navigates away from a page, tab or stepper flow before applying changes, use an Unsaved
+changes modal.
+
+![Save pattern Unsaved changes](figma:patterns-save-unsaved-changes.png)
+
+1. **Leave without saving** reverts all values on the page to the last saved state without reloading the
+   page.
+2. **Go back** returns to the previous view without discarding changes and reloading the page.
+
+### Canceling changes
+
+**Cancel** actively discards changes the user made and dismisses the current view. Cancel should never
+clear the form, or clear changes and reload the page. This increases the risk of accidental data loss.
+
+## Workflows
+
+### Basic form
+
+[Google Drive video](https://drive.google.com/file/d/1U57FoWNntJmDBPQVjx3gKZvB9EIAu2Ot/view?usp=sharing)
```

Comment:
Can we crop the final 8 seconds off this video?

Reply:
Will do 👍